### PR TITLE
LPS-169369 Replace notificationTemplateId with ERC in ObjectAction

### PR DESCRIPTION
**Pull request by guilhermedcamacho**

### CODE REVIEW

**Comment by guilhermedcamacho** [comment]
@murilostodolni , can you review my changes to see if I broke anything? Thanks!

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationTemplateLocalServiceImpl.java [hunk 1]
```diff
@@ -98,10 +102,10 @@ public NotificationTemplate addNotificationTemplate(
 		for (NotificationRecipientSetting notificationRecipientSetting :
 				notificationContext.getNotificationRecipientSettings()) {
 
-			notificationRecipientSetting.setNotificationRecipientId(
```
Comment (guilhermedcamacho) [inline modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationTemplateLocalServiceImpl.java:101]: Hey @murilostodolni , did you change this order intentionally?

Comment (murilostodolni) [reply modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationTemplateLocalServiceImpl.java:101]: Hi @guilhermedcamacho the changed was made by `source-format `

**Comment by murilostodolni** [comment]
sent to: https://github.com/liferay-objects/liferay-portal/pull/1567

**Comment by murilostodolni** [comment]
ci:close

**Pull request closed without merging**